### PR TITLE
Add documentation for cakeissues.net

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation for cakeissues.net, moved from Cake.Issues.Website repository including full history.

Part of https://github.com/cake-contrib/Cake.Issues.Website/issues/123

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Rule`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.RuleUrl`                  | Support for custom rules can be added through a custom [EsLintAddRuleUrlResolver] |
+
+[Cake.Issues.EsLint addin]: https://www.nuget.org/packages/Cake.Issues.EsLint
+[ESLint json formatter]: http://eslint.org/docs/user-guide/formatters/#json
+[EsLintAddRuleUrlResolver]: ../../../api/Cake.Issues.EsLint/EsLintIssuesAliases/D64301E6
+[EsLintJsonFormat]: ../../../api/Cake.Issues.EsLint/EsLintIssuesAliases/230C6E27
```
Comment (x-jokay) [inline docs/features.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/requirements.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.EsLint addin.
+---
+The requirements for using the [Cake.Issues.EsLint addin] are listed in the [release notes] for any specific version.
+
+[Cake.Issues.EsLint addin]: https://www.nuget.org/packages/Cake.Issues.EsLint
+[release notes]: release-notes
```
Comment (x-jokay) [inline docs/requirements.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/index.cshtml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+Title: ESLint
+Description: Issue provider which allows you to read issues logged by ESLint.
+---
+<p>@Html.Raw(Model.String(DocsKeys.Description))</p>
+
+<p>
+    Support for reading issues reported by <a href="http://eslint.org/" target="_blank">ESLint</a>
```
Comment (x-jokay) [inline docs/index.cshtml]: Use `https` for link.

File: docs/features.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Line`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Rule`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.RuleUrl`                  | Support for custom rules can be added through a custom [EsLintAddRuleUrlResolver] |
+
+[Cake.Issues.EsLint addin]: https://www.nuget.org/packages/Cake.Issues.EsLint
+[ESLint json formatter]: http://eslint.org/docs/user-guide/formatters/#json
```
Comment (x-jokay) [inline docs/features.md]: Use `https` for link.

File: docs/features.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+* Reads issues reported by ESLint.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [EsLintAddRuleUrlResolver] alias.
+
+# Supported log file formats
+
+* [EsLintJsonFormat] alias for reading issues from log files created by [ESLint json formatter].
+
+# Supported comment formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+---
+The [Cake.Issues.EsLint addin] provides the following features.
+
+# Basic features
+
+* Reads issues reported by ESLint.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [EsLintAddRuleUrlResolver] alias.
+
+# Supported log file formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

**Comment by pascalberger** [comment]
@x-jokay Ready for another review.

Top level headings are  required to have the entry appear in site TOC (See https://github.com/cake-contrib/Cake.Issues.Website/pull/41)

**Pull request merged**